diff --git a/components/webui/flags/pref_service_flags_storage.cc b/components/webui/flags/pref_service_flags_storage.cc
index dda7d5c534..422cf0c0a6 100644
--- a/components/webui/flags/pref_service_flags_storage.cc
+++ b/components/webui/flags/pref_service_flags_storage.cc
@@ -87,8 +87,18 @@ void PrefServiceFlagsStorage::CommitPendingWrites() {

 // static
 void PrefServiceFlagsStorage::RegisterPrefs(PrefRegistrySimple* registry) {
-  registry->RegisterListPref(prefs::kAboutFlagsEntries);
-  registry->RegisterDictionaryPref(prefs::kAboutFlagsOriginLists);
+  base::ListValue default_flags;
+  default_flags.Append("extension-mime-request-handling@2");
+  default_flags.Append("enable-parallel-downloading@1");
+  default_flags.Append("smooth-scrolling@1");
+  default_flags.Append("custom-ntp");
+  default_flags.Append("close-window-with-last-tab@1");
+  registry->RegisterListPref(prefs::kAboutFlagsEntries, std::move(default_flags));
+
+  base::DictValue default_origin_lists;
+  default_origin_lists.Set("custom-ntp", "chrome://new-tab-page");
+  registry->RegisterDictionaryPref(prefs::kAboutFlagsOriginLists,
+                                   std::move(default_origin_lists));
 }

 #if BUILDFLAG(IS_CHROMEOS)
