diff --git a/components/webui/flags/pref_service_flags_storage.cc b/components/webui/flags/pref_service_flags_storage.cc
index dda7d5c534..50f69d05d3 100644
--- a/components/webui/flags/pref_service_flags_storage.cc
+++ b/components/webui/flags/pref_service_flags_storage.cc
@@ -87,7 +87,10 @@ void PrefServiceFlagsStorage::CommitPendingWrites() {
 
 // static
 void PrefServiceFlagsStorage::RegisterPrefs(PrefRegistrySimple* registry) {
-  registry->RegisterListPref(prefs::kAboutFlagsEntries);
+  base::ListValue default_flags;
+  default_flags.Append("extension-mime-request-handling@2");
+  default_flags.Append("enable-parallel-downloading@1");
+  registry->RegisterListPref(prefs::kAboutFlagsEntries, std::move(default_flags));
   registry->RegisterDictionaryPref(prefs::kAboutFlagsOriginLists);
 }
 
