diff --git a/chrome/browser/resources/settings/about_page/about_page.html.ts b/chrome/browser/resources/settings/about_page/about_page.html.ts
--- a/chrome/browser/resources/settings/about_page/about_page.html.ts
+++ b/chrome/browser/resources/settings/about_page/about_page.html.ts
@@ -117,9 +117,6 @@ export function getHtml(this: SettingsAboutPageElement) {
       <div class="secondary">$i18n{aboutProductCopyright}</div>
     </div>
 
-    <div class="info-section">
-      <div class="secondary">$i18nRaw{aboutProductLicense}</div>
-    </div>
 <if expr="_google_chrome or _is_chrome_for_testing_branded">
     <div class="secondary">
       <a id="tos" href="$i18n{aboutTermsURL}">$i18n{aboutProductTos}</a>
diff --git a/chrome/browser/ui/webui/settings/settings_localized_strings_provider.cc b/chrome/browser/ui/webui/settings/settings_localized_strings_provider.cc
--- a/chrome/browser/ui/webui/settings/settings_localized_strings_provider.cc
+++ b/chrome/browser/ui/webui/settings/settings_localized_strings_provider.cc
@@ -351,12 +351,6 @@ void AddAboutStrings(content::WebUIDataSource* html_source, Profile* profile) {
           l10n_util::GetStringUTF16(IDS_ABOUT_VERSION_COPYRIGHT),
           base::Time::Now()));
 
-  std::u16string license = l10n_util::GetStringFUTF16(
-      IDS_VERSION_UI_LICENSE, chrome::kChromiumProjectURL,
-      chrome::kChromeUICreditsURL16,
-      l10n_util::GetStringUTF16(IDS_SETTINGS_OPENS_IN_NEW_TAB));
-  html_source->AddString("aboutProductLicense", license);
-
   html_source->AddBoolean("aboutObsoleteNowOrSoon",
                           ObsoleteSystem::IsObsoleteNowOrSoon());
   html_source->AddBoolean("aboutObsoleteEndOfTheLine",
